src/pages/Clusters.js

```diff
@@ -56,7 +55,7 @@ const Clusters = () => {
   const clustersByProject = _.groupBy('googleProject', clusters)
 
   return h(Fragment, [
-    h(TopBar, { title: 'Notebook runtimes', href: Nav.getLink('clusters') }),
+    h(TopBar, { title: 'Notebook runtimes' }),
```

Comment:
Removing this href allows the Terra logo to redirect to the home page from the notebook runtimes view